app/authentication/auth.py

```diff
@@ -195,15 +196,33 @@ def validate_service_api_key_auth():  # noqa: C901
             err_msg = 'Error: Your system clock must be accurate to within 30 seconds'
             raise AuthError(err_msg, 403, service_id=service.id, api_key_id=api_key.id)
 
-        if api_key.expiry_date:
+        # The revoked field is added as a temporary measure until we can implement proper use of the expiry date
```

Comment:
is there a ticket for this work? if so, please include it as a TODO on this line. If not, make a ticket?

Reply:
I don't think there's a ticket, but I'll check, or make one if necessary.